WebJobs.Script.sln

```diff
@@ -160,6 +160,49 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "BlobTrigger-Batch", "BlobTr
 		sample\BlobTrigger-Batch\run.bat = sample\BlobTrigger-Batch\run.bat
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "EasyTables", "EasyTables", "{3EACC3F3-2747-42D6-B826-6A150F9EA92A}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "CSharp", "CSharp", "{9A295BFF-005D-456B-BBEA-6F4F389B7A66}"
+	ProjectSection(SolutionItems) = preProject
+		sample\EasyTables\CSharp\host.json = sample\EasyTables\CSharp\host.json
```

Comment:
The samples already have a host file - not sure what this is doing here. Perhaps you're trying to follow the pattern I use in the e2e tests? This samples directory isn't intended for tests - recommend moving all your test samples to the test directory, and keeping just 2-3 representative samples in this directory.